Source/WebKit/UIProcess/API/Cocoa/_WKSystemPreferencesInternal.h

```diff
@@ -27,6 +27,10 @@
 #import <wtf/text/ASCIILiteral.h>
 
 const auto LDMEnabledKey = CFSTR("LDMGlobalEnabled");
+
+// This string must remain consistent with the CFString value below it.
 constexpr auto WKLockdownModeEnabledKey = "WKLockdownModeEnabled"_s;
+const auto WKLockdownModeEnabledKeyCFString = CFSTR("WKLockdownModeEnabled");
```

Comment:
Since consistency is the only invariant here. Have you considered something like?

```cpp
#define WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO WKLockdownModeEnabled
constexpr auto WKLockdownModeEnabledKey = STRINGIZE_VALUE_OF(WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO)_s;
const auto WKLockdownModeEnabledKeyCFString = CFSTR(STRINGIZE_VALUE_OF(WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO));
```

Reply:
Not compiling.

Reply:
Ah that's sad. I looked at the preprocessor output. The error is because of the extra space.

```cpp
const auto WKLockdownModeEnabledKeyCFString = "WKLockdownModeEnabled" _s;
```

Reply:
@RSilicon this works and doesn't introduce any behavior change.

```cpp
#define WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO WKLockdownModeEnabled
constexpr auto WKLockdownModeEnabledKey = WTF_CONCAT(STRINGIZE_VALUE_OF(WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO), _s);
const auto WKLockdownModeEnabledKeyCFString = CFSTR(STRINGIZE_VALUE_OF(WK_LOCKDOWN_MODE_ENABLED_KEY_MACRO));
```